express.js

```diff
@@ -238,23 +239,25 @@ let getLatestClientRelease = require("./scripts/getLatestClientRelease");
 extractor.run();
 getLatestClientRelease.run();
 
-// Run leaderboard extractor every minute
-setTimeout(() => {
+// Run leaderboard extractor every 15 minutes
+schedule.scheduleJob('*/15 * * * *', function myFunc() {
```

Comment:
An anonymous function is adequate in this case. The same applies to the other scheduleJob.

Reply:
u mean an arrow function?
I am sorry I got not much of an idea if it comes to javascript?

Reply:
No worries. Just use function() so it looks like this `schedule.scheduleJob('*/15 * * * *', function() {`.

Reply:
oh ok